canonical/lxd-ui
# Factor out iso selection on instance launch

## Done

- extracted dedicated list for iso files when picking an image for the instance launch

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - create instance > browse images > use custom iso > upload custom iso
    - ensure the instance creation flow with uploading or selecting a custom iso file works flawless
